resources/views/entities/components/header_grid.blade.php

```diff
@@ -287,3 +288,15 @@
         </article>
     </dialog>
 @endsection
+
+
+@section('styles')
+    @parent
+    <style>
+        /** Entity Images URL**/
+        :root {
+            --entity-image-url: {{$imageUrl}};
```

Comment:
Since these are strings, they need to be surrounded by ', so '{{ $imageUrl }}` (don't forget space either)